x-pack/plugins/security/server/config.test.ts

```diff
@@ -150,31 +153,23 @@ describe('config schema', () => {
   });
 
   it('should throw error if xpack.security.encryptionKey is less than 32 characters', () => {
-    expect(() =>
-      ConfigSchema.validate({ encryptionKey: 'foo' })
-    ).toThrowErrorMatchingInlineSnapshot(
-      `"[encryptionKey]: value has length [3] but it must have a minimum length of [32]."`
+    expect(() => ConfigSchema.validate({ encryptionKey: 'foo' })).toThrow(
+      '[encryptionKey]: value has length [3] but it must have a minimum length of [32].'
```

Comment:
Yeh, this was because of the multi line issue with jest error snapshots. 
The single line error snapshots worked fine but I thought it's better if they're consistent